examples/visualization/brain.py

```diff
@@ -0,0 +1,106 @@
+"""
+.. _ex-brain:
+
+===============================
+Plotting with ``mne.viz.Brain``
+===============================
+
+In this example, we'll show how to use :class:`mne.viz.Brain`.
+"""
+# Author: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+# Plot a brain image
+# ------------------
+#
+# First we plot a brain. This requires a directory with MRI? information.
+# MNE provides functionality to automatically retrieve standard head images.
+# In this example we use the ``sample`` data which is data from an example
+# subject being shown auditory and visual stimuli.
+
+import os.path as op
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+brain = mne.viz.Brain('sample', subjects_dir=subjects_dir,
```

Comment:
What is sample? Is there an existing MNE doc page we can link to that describes this?

The tutorials seem to switch between fsaverage and sample. But I can't figure out why some use one and some use another. I think this is confusing and make copying and pasting parts from multiple tutorials risky.

Are the two terms interchangeable, but you just need to be consistent? Can we add some text to that effect?

Reply:
`sample` is the subject from the sample dataset in MNE, `fsaverage` is a template space that can be linearly registered to using the Talairach transform or non-linearly registered to using Freesurfer's combined surface volumetric warp or symmetric diffeomorphic registration (that's what MNE uses) see https://mne.tools/dev/auto_tutorials/clinical/10_ieeg_localize.html for some more explanation

Reply:
I think as this is the Brain explain location we should add a brief explanation here. Or link to a specific subsection in other page.

Reply:
> that's what MNE uses

If MNE uses fsaverage then why does this tutorial use something else? As this is a learning resource I think it should demonstrate the canonical usage. And I'd guess most users don't have their own MRI scans.

Reply:
MNE uses symmetric diffeomorphic registration to register `sample` to `fsaverage` to take an individual subject's scan and morph it to a template (`fsaverage`).

So to answer your question MNE uses `sample` for individual-level results and `fsaverage` for group results in general (there may be exceptions to this).

Reply:
Wonderful. That's a great insight. I'm sure there will be exceptions to the rule of thumb. But great to know.

Reply:
As a final suggestion then. Should we use fsaverage for the example then? As most users would not have individual subject MRIs. And then a final section to the tutorial that demonstrates individual subject scans are simply a case of changing to `sample`?

Reply:
If you want to perform source estimation, you need an accurate head model, hence you need the subject's MRI. While it is technically possible to use `fsaverage` instead, this gives poor results. IMO, we should carry the message in the tutorials that when you are doing source estimation and visualizing it and stuff, **of course** you have the individual's MRI at hand. Not having it should be an absolute exception.

Reply:
Hmmm I've heard from Mark George's group at MUSC that this worked pretty well for TMS-EEG localization comparing known subject-specific MRI to template but I haven't tried first hand... We have fsaverage and sample's MRI maybe we should plot both and comment on how good or bad it is and maybe cite relevant sources